include/avk/semaphore.hpp

```diff
@@ -48,6 +48,32 @@ namespace avk
 		const auto& handle() const { return mSemaphore.get(); }
 		const auto* handle_addr() const { return &mSemaphore.get(); }
 
+		// timeline semaphore specific functions
+
+		// returns the current value of the timeline semaphore
+		const uint64_t query_current_value() const;
+		// sets the timeline semaphore to the specified value

```

Comment:
Please consistently use `/** ... */` comments for all the public methods!

Reply:
Please also use new lines to separate two methods.

Reply:
✅